sparse_strips/vello_common/src/glyph.rs

```diff
@@ -140,20 +135,20 @@ impl<'a, T: GlyphRenderer + 'a> GlyphRunBuilder<'a, T> {
                 continue;
             }
 
-            let mut local_transform =
-                Affine::translate(Vec2::new(glyph.x as f64 * scale, glyph.y as f64 * scale));
-            if let Some(skew) = horizontal_skew {
-                local_transform *= Affine::skew(skew.tan() as f64, 0.0);
-            }
-            if let Some(glyph_transform) = glyph_transform {
-                local_transform *= glyph_transform;
+            let mut translation = translation_transform
+                .pre_translate(Vec2::new(glyph.x as f64, glyph.y as f64))
+                .translation();
```

Comment:
With proper inlining, the compiler should be able to elide most float calculations here, but I've not verified that. We could also instead store the translation transformation affine as a 2x2 matrix instead of the full 3x2 `Affine` (the translation transform is just the global transform with translation set to 0), and manually calculate the following.

```
translate_x = a * glyph.x + c * glyph.y
translate_y = b * glyph.x + d * glyph.y
```

Thoughts?

Reply:
Actually, we can take the global transform from the run.

```rust
let [a, b, c, d, _, _] = self.run.transform.as_coeffs();
let mut translation = Vec2::new(
    a * glyph.x as f64 + c * glyph.y as f64,
    b * glyph.x as f64 + d * glyph.y as f64,
);
```

Reply:
I've applied this in 1b51ec0112f34528ea1032d43f14e4a864150aca.